app/services/folio_client.rb

```diff
@@ -155,8 +155,11 @@ def authenticated_request(path, headers: {}, **other)
   end
 
   def request(path, headers: {}, method: :get, **other)
+    timeout = other[:timeout] || Settings.folio.timeout
+    other.delete(:timeout)
```

Comment:
```suggestion
  def request(path, headers: {}, method: :get, timeout: Settings.folio.timeout, **other)
```

Is there a reason we want to pass timeout to the `HTTP#request` method?

Reply:
Sorry, I'm not following. Isn't the timeout kwarg already flowing through those methods?

Reply:
Maybe I'm missing something, but I still don't think it needs to move? Here's what I'm seeing:

```
def authenticated_request(**other)
  request(**other)
end

def request(timeout: -1)
  puts timeout
end
```

```
authenticated_request(timeout: 5)
# => prints 5

authenticated_request
# => prints -1
```

Is there something else going on here?